src/Sylius/Bundle/CoreBundle/Listener/CatalogPromotionCreatedListener.php

```diff
@@ -14,17 +14,17 @@
 namespace Sylius\Bundle\CoreBundle\Listener;
 
 use Doctrine\ORM\EntityManagerInterface;
-use Sylius\Bundle\CoreBundle\Processor\AllCatalogPromotionsProcessorInterface;
+use Sylius\Bundle\CoreBundle\Processor\RequestProductVariantCatalogPromotionRecalculateInterface;
 use Sylius\Component\Core\Model\CatalogPromotionInterface;
 use Sylius\Component\Promotion\Event\CatalogPromotionCreated;
 use Sylius\Component\Resource\Repository\RepositoryInterface;
 
 final class CatalogPromotionCreatedListener
 {
     public function __construct(
-        private AllCatalogPromotionsProcessorInterface $allCatalogPromotionsProcessor,
-        private RepositoryInterface $catalogPromotionRepository,
-        private EntityManagerInterface $entityManager
+        private RequestProductVariantCatalogPromotionRecalculateInterface $allCatalogPromotionsProcessor,
+        private RepositoryInterface                                       $catalogPromotionRepository,
+        private EntityManagerInterface                                    $entityManager
```

Comment:
You should revert the arguments aligning here and in other places
```suggestion
        private RepositoryInterface $catalogPromotionRepository,
        private EntityManagerInterface $entityManager
```